sdv/datasets/demo.py

```diff
@@ -122,6 +124,18 @@ def _get_metadata(output_folder_name, in_memory_directory, dataset_name):
     return metadata
 
 
+@lru_cache()
+def _download_demo(modality, dataset_name, output_folder_name=None):
```

Comment:
instead of caching here, can we make a download_test_demo fixture in our test utils and just cache that one?